docs/examples/101/azure-spring-cloud/activedeployment.bicep

```diff
@@ -0,0 +1,27 @@
+param app1Name string
+param app2Name string
+param app3Name string
+
+resource app1 'Microsoft.AppPlatform/Spring/apps@2020-07-01' = {
+  name: app1Name
```

Comment:
similar question on naming here. Does this assume a parent `Spring` resource exists? And does the name not need to be prefixed by the name of the `Spring` resource?

Reply:
This is the case. Check the content of the variable I pass on main.bicep. I tested the deployment template with success because I had a doubt on it btw.

Reply:
Whoops! Sorry, for some reason, I missed that this was an example with modules. Thanks for the clarification.